src/theme/default/components/gridblock/sanity-query.ts

```diff
@@ -50,11 +50,103 @@ export function query(pageAndComponentCombo: PageAndSingleComponentDetails){
           __typename
           componentsGrid{
               __typename
+              ... on TextBlock {
```

Comment:
These subcomponent lookups will be done in the components own query file.  Via loadSingleComponentGraphQLData